specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2020-07-01-preview/examples/CreateExtension.json

```diff
@@ -0,0 +1,61 @@
+{
+  "parameters": {
+    "subscriptionId": "subId1",
+    "resourceGroupName": "rg1",
+    "clusterRp": "Microsoft.Kubernetes",
+    "clusterResourceName": "connectedClusters",
+    "extensionInstanceName": "ClusterMonitor",
+    "api-version": "2020-07-01-preview",
```

Comment:
@NarayanThiru and @safeermohammed - As there are multiple commits in this PR, its really hard for me to look at the changes. For faster review, can you please send a new PR? To easily identify the delta changes from previous api version, the recommendation is to copy the existing version into new directory structure for first commit and then push new changes in subsequent commits.